test/test_epoll.cpp

```diff
@@ -0,0 +1,142 @@
+#include "gtest/gtest.h"
+#include "api.h"
+#include "epoll.h"
+
+
+TEST(CEPoll, InfiniteWait)
+{
+	ASSERT_EQ(srt_startup(), 0);
+
+	const int epoll_id = srt_epoll_create();
+	ASSERT_GE(epoll_id, 0);
+
+	ASSERT_EQ(srt_epoll_wait(epoll_id, nullptr, nullptr,
+		nullptr, nullptr,
```

Comment:
Google Test requires a C++11-standard-compliant compiler, so in the unit-testing part, I would suggest allowing the usage of C++11.

Reply:
The version of Google Test we currently use (1.8.x) specifically does not require C++11 compliant compiler. So, in effect, here we lose C++03 compatibility because of one's preference for `nullptr` vs `NULL`.